### ISSUE TITLE
Problem loading assets from dev server on windows

### ISSUE THREAD

**Issue by meypod** [callstack/repack#247]
Currently there's a serious problem on windows, which took me hours to track down and do a local fix

**tl;dr**: made the issue to create a PR fixing this.

The issue is that `filename` in [assetsCache](https://github.com/callstack/repack/blob/8480e2cb878c56bea5d6adf13ded5e579b300a2c/packages/repack/src/webpack/Compiler.ts#L113) in windows uses backslashes. when the app tries to load the asset, it requests the localhost server using forward slashes, so it will miss. my quick solution to this was adding a duplicate key with slashes replaced by forward slashes and pointing to the same data. duplication was done because I do not know if the `assetsCache` keys are used inside production builds or not. since removing the backslashed paths would cause local file system access failure, I did not remove it. it will also simply replace the other key on unix paths.
